optimum/exporters/onnx/convert.py

```diff
@@ -1196,6 +1201,18 @@ def onnx_export_from_model(
         optimization_config.disable_shape_inference = True
         optimizer.optimize(save_dir=output, optimization_config=optimization_config, file_suffix="")
 
+    if slim:
+        if not is_onnxslim_available():
+            raise ImportError("The pip package `onnxslim` is required to optimize onnx models.")
+
+        from onnxslim import slim
+
+        onnx_models = [os.path.join(output, x) for x in os.listdir(output) if x.endswith(".onnx")]
```

Comment:
here you should probably do soemthing like glob(*.onnx) because in some cases (diffusers for example) the onnx files won't be directly in the folder `output`, but rather in subfolders like `output/unet/model.onnx` etc